kms-message/src/kms_crypto_apple.c

```diff
@@ -58,4 +61,83 @@ kms_sha256_hmac (void *unused_ctx,
    return true;
 }
 
+static void
+safe_CFRelease (CFTypeRef ptr)
+{
+   if (ptr) {
+      CFRelease (ptr);
+   }
+}
+
+bool
+kms_sign_rsaes_pkcs1_v1_5 (void *unused_ctx,
+                           const char *private_key,
+                           size_t private_key_len,
+                           const char *input,
+                           size_t input_len,
+                           unsigned char *signature_out)
+{
+   CFDataRef key_data_ref = NULL;
+   CFDataRef pass_ref = NULL;
+   SecItemImportExportKeyParameters import_params;
+   OSStatus status;
+   /* TODO: I think the expected format should be kSecFormatWrappedPKCS8, but
+    * GCP keys appear to only load for kSecFormatBSAFE. */
+   SecExternalFormat format = kSecFormatUnknown;
```

Comment:
I'm still not sure what the deal is here. I left this as `kSecFormatUnknown` for now.

Reply:
Per slack discussion, `kSecFormatWrappedPKCS8` is distinct from `PKCS8`.